### ISSUE TITLE
Gossamer generated raw spec with empty BABE authorities

### ISSUE THREAD

**Issue by EclesioMeloJunior** [ChainSafe/gossamer#2589]
## Describe the bug

Using the same human-readable specification such as https://gist.githubusercontent.com/danforbes/b246640cbedd1107ff412d5d25851058/raw/815fa77feee103e518a41a3f593c5d4442d14db6/gistfile1.txt gives an empty BABE authority set.

Running the following commands the gossamer node will start but will not produce blocks because the current BABE authority set will be empty.
```
gossamer build-spec --genesis-spec ./human-readable.json --raw --output ./spec.json
```
And then
```
gossamer --chain ./spec.json
```

However, using the Polkadot binary to generate the raw specification from the human-readable one and using the raw spec to start a gossamer node the BABE authority set will contain authorities.

**Comment by timwu20** [comment]
Let's de-prioritise this for now and focus on supporting the latest runtime.  Let's consider removing this functionality entirely and have a discussion about it.

**Comment by danforbes** [comment]
Is this essentially a duplicate of #2417?

**Comment by EclesioMeloJunior** [comment]
No, this is related to a likely problem while building the key value map with infos from the json spec

**Comment by danforbes** [comment]
Iceboxing this because we will probably de-prioritize our ability to generate/build specs for now.
